src/Pages/Rules/rules.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Rules = () => {
+  return (
+    <section className="hero">
+      <div
+        className="hero-left"
+        style={{ marginLeft: "5rem", marginTop: "-1rem" }}
+      >
+        <h1>Rules and Regulations</h1>
```

Comment:
Suggestion: you can create JSON data for rules and through map, you can import data here

Reply:
Thankyou for your suggestions will surely implement that.